acl_loader/main.py

```diff
@@ -651,7 +661,17 @@ def convert_rule_to_db_schema(self, table_name, rule):
         rule_props["PRIORITY"] = str(self.max_priority - rule_idx)
 
         # setup default ip type match to dataplane acl (could be overriden by rule later)
-        if self.is_table_l3v6(table_name):
+        if self.is_table_l3v4v6(table_name):
+            # ETHERTYPE must be passed and it should be one of IPv4 or IPv6
+            try:
+                ether_type =  rule.l2.config.ethertype
+            except Exception as e:
```

Comment:
Why don't write the `ether_type` into `rule_props` as we do for `L3` and `L3V6`?
And can we use `ETHERTYPE_IPV6` for IPv6 rules?